packages/server-side-render/src/server-side-render.js

```diff
@@ -39,11 +39,22 @@ function DefaultErrorResponsePlaceholder( { response, className } ) {
 	return <Placeholder className={ className }>{ errorMessage }</Placeholder>;
 }
 
-function DefaultLoadingResponsePlaceholder( { className } ) {
+function DefaultLoadingResponsePlaceholder( { className, children } ) {
 	return (
-		<Placeholder className={ className }>
-			<Spinner />
-		</Placeholder>
+		<div className={ className } style={ { position: 'relative' } }>
```

Comment:
@Mamaduka The reason why I removed the `className` reference here is because it also is included here: https://github.com/WordPress/gutenberg/pull/28289/files#diff-1fcc38b89edc0d5f7b6239481f42d74b8e2010d792fb26b1e0f8623b222a06fbR160 meaning we now have markup where we nest the className in itself. And I worried that this would lead to stying conflicts. 

Should we maybe do something like `${className}__loading-placeholder` ?

Reply:
I a good call, and sorry I missed the obvious clue 😄 

The `DefaultLoadingResponsePlaceholder` also received a complete set of props, so if someone is overriding the default placeholder, this change won't break anything.

I'm going to revert the change.